templates/microcloud.html

```diff
@@ -0,0 +1,249 @@
+{% extends "_base/base.html" %}
+{% set active_page = "microcloud" %}
+
+
+{% block title %}MicroCloud{% endblock %}
+
+{% block meta_copydoc
+%}https://docs.google.com/document/d/1KzG5qScbehgotJYYWBDM82-9BnUN7mc4Omg9Sa2Nehg/edit#heading=h.atgwxemoxr48{% endblock
+meta_copydoc %}
+{% block body_class %}is-paper{% endblock %}
+
+{% block outer_content %}
+<section class="p-section--hero">
+    <div class="row--50-50">
+        <div class="col">
+            <h1>MicroCloud</h1>
+        </div>
+        <div class="col">
+            <div class="p-image-container--16-9 is-highlighted u-hide--small">
+                <img src="https://assets.ubuntu.com/v1/43d1d899-Canonical-MicroCloud.png" style="padding: 1rem;" alt="" />
+            </div>
+            <p>MicroCloudで有効なエッジコンピューティングを計画</p>
+            <hr class="is-muted" />
+            <p>
+                <a class="p-button" href="/engage/guide-to-micro-clouds-jp">MicroCloudガイドを読む</a>
+            </p>
+        </div>
+    </div>
+</section>
+
+<section class="p-section">
+    <div class="row--50-50">
+        <hr />
+        <div class="col">
+            <h2>MicroCloudとは</h2>
+        </div>
+        <div class="col">
+            <p>MicroCloudとは、エッジでのオンデマンドコンピューティングに適した軽量、多機能でシンプルなクラウドです。IoTは多数の単体マシンやセンサーを使用してデータを収集し、コンピューティングタスクは実行しません。マイクロクラウドは違います。MicroCloudは、無人動作、自律動作、クラスタリングの機能を備えた実績あるクラウドプリミティブを再利用し、エッジコンピューティングの一般的な問題を解決します。
+            </p>
+            <p>MicroCloudは、消費者がどこにいてもその近くでコンピューター処理を実行し、まさに必要な場所でクラウドの威力を提供します。MicroCloudはもともと手軽であり、システム3つしか必要としません。サイトの規模は簡単に調節でき、3ノードから50ノードのクラスターにまで拡張できます。MicroCloudは、LXD、Ceph、OVNというトップクラスのオープンソースコンポーネントを含みます。
+            </p>
+        </div>
+    </div>
+</section>
+
+<section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+        <hr />
+        <h2>短時間でクラウドの運用を開始</h2>
+    </div>
+    <div class="row--25-75-on-large">
+        <div class="col">
+            <div class="p-equal-height-row has-divider-1">
+                <div class="p-equal-height-row__col">
+                    <div class="p-equal-height-row__item">
```

Comment:
Here in we can use `u-hide-small` to hide the image and not disturb other formats